src/main/app/claims/claimStoreClient.ts

```diff
@@ -171,4 +181,21 @@ export class ClaimStoreClient {
       .get(`${claimStoreApiUrl}/${reference}/defendant-link-status`)
       .then(linkStatus => linkStatus.linked)
   }
+
+  retrieveUserRoles (user: User): Promise<string> {
```

Comment:
I would prefer list to be returned not string. If somebody needs string it is easy to join it together. Splitting is more complicated. Since we have well structured data why to make it a string. Please change to list.